pkg/yang/entry.go

```diff
@@ -1482,25 +1480,18 @@ func (s sortedErrors) Less(i, j int) bool {
 		return false
 	}
 
-	// compare compares field x to see which is less.
```

Comment:
Was this unused?

Reply:
This is used, I simplified it and took it out of the function.